content/1.7/back-office/quick-access/add-edit-quick-access.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Add / edit quick access
+weight: 1
+---
+
+# Add / edit quick access
+
+The following fields/toggles are displayed:
+
+- **Name**: this field is mandatory for the default language. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "The maximum allowed characters is 32.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. The field has a character limit of 32, it does not allow to enter more of them.
```

Comment:
```suggestion
- **Name**: this field is mandatory for the default language. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "Forbidden characters: <>;=#{}". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with the default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. The field has a character limit of 32, it does not allow to enter more characters.
```

Reply:
What's happening if I enter more than 32 characters and save, is there an error ?